tests/rolling-upgrade-gce.yaml

```diff
@@ -6,14 +6,16 @@ n_monitor_nodes: 1
 gce_datacenter: 'us-east1-b'
 
 # workloads
+write_stress_during_entire_test: cassandra-stress write no-warmup cl=QUORUM duration=80m -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..10000000000000 -log interval=5
 prepare_write_stress: cassandra-stress write no-warmup cl=QUORUM n=10000000 -schema 'replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..10000000 -log interval=5
 stress_cmd_read_10m: cassandra-stress read no-warmup cl=QUORUM duration=10m -schema 'replication(factor=3) compression=SnappyCompressor' -port jmx=6868 -mode cql3 native compression=snappy -rate threads=1000 -pop seq=1..10000000 -log interval=5
-stress_cmd_read_clall: cassandra-stress read no-warmup cl=ALL n=10000000 -schema 'replication(factor=3) compression=DeflateCompressor' -port jmx=6868 -mode cql3 native compression=none -rate threads=1000 -pop seq=1..10000000 -log interval=5
+stress_cmd_read_cl_one: cassandra-stress read no-warmup cl=ALL n=10000000 -schema 'replication(factor=3) compression=DeflateCompressor' -port jmx=6868 -mode cql3 native compression=none -rate threads=1000 -pop seq=1..10000000 -log interval=5
 stress_cmd_read_20m: cassandra-stress read no-warmup cl=QUORUM duration=20m -schema 'replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..10000000 -log interval=5
-stress_cmd_sst3_prepare: cassandra-stress user no-warmup profile=/tmp/sst3_schema.yaml  ops'(insert=1)' cl=QUORUM n=10000000 -rate threads=1000 -pop seq=1..10000000
-stress_cmd_sst3_verify_read: cassandra-stress user no-warmup profile=/tmp/sst3_schema.yaml  ops'(read1=1)' cl=ALL n=10000000 -rate threads=1000 -pop seq=1..10000000
-stress_cmd_sst3_verify_more: cassandra-stress user no-warmup profile=/tmp/sst3_schema.yaml  ops'(read1=1,read2=1,update_static=1,update_ttl=1,update_diff1_ts=1,update_diff2_ts=1,update_same1_ts=1,update_same2_ts=1)' cl=ALL n=10000000 -rate threads=200 -pop seq=1..10000000
-stress_cmd_sst3_verify_delete: cassandra-stress user no-warmup profile=/tmp/sst3_schema.yaml  ops'(delete_row=1)' cl=ALL n=1000000 -rate threads=200 -pop seq=1..1000000
+verify_stress_after_cluster_upgrade: cassandra-stress read no-warmup cl=ONE duration=80m -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..10000000000000 -log interval=5
```

Comment:
1. Shouldn't be duration, but exact number like in the first step.
2. Change the name to be verify_data_after_entire_test